### ISSUE TITLE
Cannot build on MacOS Catalina

### ISSUE THREAD

**Issue by f0rb1dd3n** [jollheef/out-of-tree#19]
I could not build out-of-tree in MacOS Catalina. What can I do?

```
$ go get -u code.dumpstack.io/tools/out-of-tree
$ cd $GOPATH/src/code.dumpstack.io/tools/out-of-tree/examples/kernel-module
~/src/code.dumpstack.io/tools/out-of-tree/examples/kernel-module$ out-of-tree kernel autogen                                                                                                master
main.go:249: No ~/.out-of-tree/kernels.toml: Probably you need to run `out-of-tree kernel autogen` in directory that contains .out-of-tree.toml with defined kernel masks (see docs at https://out-of-tree.io)
main.go:276: open /Users/ighor/.out-of-tree/kernels.toml: no such file or directory
kernel.go:548: Generating for kernel mask {Ubuntu 16.04 4[.]4[.]0-70-.* {[] [] [] []}}
kernel.go:380: out_of_tree_ubuntu_16__04.img not exists, start downloading...
kernel.go:128: Base image for Ubuntu:16.04 not found, start generating
main.go:351: open /proc/self/maps: no such file or directory
```

**Comment by jollheef** [comment]
Hello,

Thanks for the bug report. I'll try to investigate.

Can you provide more info? The exact version of OS, out-of-tree and docker.

Can you also check that docker works for sure? (`docker run hello-world`)

**Comment by f0rb1dd3n** [comment]
Hello,

Thanks for answering me. So, the informations follow below:

MacOS Catalina Version: 10.15.1
Darwin Kernel Version: 19.0.0

out-of-tree version: latest (I just tried to download and run).

docker:
```
$ docker run hello-world
Unable to find image 'hello-world:latest' locally
latest: Pulling from library/hello-world
1b930d010525: Pull complete
Digest: sha256:4df8ca8a7e309c256d60d7971ea14c27672fc0d10c5f303856d7bc48f8cc17ff
Status: Downloaded newer image for hello-world:latest

Hello from Docker!
This message shows that your installation appears to be working correctly.

To generate this message, Docker took the following steps:
 1. The Docker client contacted the Docker daemon.
 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
    (amd64)
 3. The Docker daemon created a new container from that image which runs the
    executable that produces the output you are currently reading.
 4. The Docker daemon streamed that output to the Docker client, which sent it
    to your terminal.

To try something more ambitious, you can run an Ubuntu container with:
 $ docker run -it ubuntu bash

Share images, automate workflows, and more with a free Docker ID:
 https://hub.docker.com/

For more examples and ideas, visit:
 https://docs.docker.com/get-started/

$ docker --version
Docker version 19.03.5, build 633a0ea
```

**Comment by jollheef** [comment]
Finally, I got a macOS machine (yes, that's why it took so long) to reproduce this issue.

Fixed in v1.2.1.

**Comment by f0rb1dd3n** [comment]
It is working good. Thank you!
